app/src/App.js

```diff
@@ -118,5 +126,12 @@ const App = () => {
     return <button onClick={() => loginWithRedirect()}>Log In</button>;
   }
 };
+const DatePickerComponent = () => {
+  const [startDate, setStartDate] = React.useState(new Date());
+  console.log(startDate);
+  return (
+    <DatePicker selected={startDate} onChange={(date) => setStartDate(date)} />
```

Comment:
Date Picker component seems to be working properly! I am able to click the date and a calendar pops up and I am able to select what I want